api/PaymentRequest.json

```diff
@@ -276,7 +276,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "11.1"
+              "version_added": "12.1"
```

Comment:
I've confirmed support with https://mdn-bcd-collector.appspot.com/tests/api/PaymentRequest/merchantvalidation_event in Safari 11.1 on BrowserStack. Making an edit like this which is contradicted by collector results will likely result in it surfacing again later, and likely making the change without noticing it's inconsistent with `MerchantValidationEvent`.